examples/extract-auth-token.py

```diff
@@ -4,12 +4,44 @@
 import libflagship.logincache
 import json
 import sys
+import platform
+import os
+import getopt
 
-if len(sys.argv) != 2:
-    print(f"usage: {sys.argv[0]} <path/to/login.json>")
-    exit(1)
+def jankauto():
+    userdir = os.getlogin()
+    useros = platform.system()
 
-data = open(sys.argv[1]).read()
+    darfileloc = f'/Users/{userdir}/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json'
+    winfileloc = os.path.join('C:\\Users\\') + f'{userdir}' + os.path.join('\\AppData\\Local\\Ankermake\\login.json')
```

Comment:
We can do this slightly cleaner. On windows, we can use `%APPDATA%` to refer specifically to the user's appdata dir:

```python
from os import path
...
win_path = path.expandvars(r'%LOCALAPPDATA%\Ankermake\login.json')
# or (possibly more compatible, not sure)
win_path = path.expandvars(r'%APPDATA%\Local\Ankermake\login.json')
```

For mac, we could do:

```python
mac_path = path.expandhome("~/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json")
```

These are built into python, but maybe something like [platformdirs](https://pypi.org/project/platformdirs/) might be even smoother?